dockerfile_functions.sh

```diff
@@ -76,6 +76,25 @@ print_ubi-minimal_ver() {
 	EOI
 }
 
+print_centos_ver() {
+	os_version="7"
```

Comment:
I'd suggest using `cent os 8` as that is already available. Any reason why we want to use the older version ?

Reply:
I can bump it up no problem, but ClefOS will not be available for a while. See @nealef response above. 

Let me know if you want me to keep them in sync or have CentOS on 8 and ClefOS on 7.